### PR TITLE
chore(rxStatusColumn): decompose monolithic source

### PR DESCRIPTION
**Pull request by CITguy**
JIRA: https://jira.rax.io/browse/FRMW-365

### LGTMS
- [x] Dev LGTM
- [x] Dev LGTM

### CODE REVIEW

File: src/rxStatusColumn/scripts/rxStatusMappings.js [hunk 1]
```diff
@@ -1,177 +1,4 @@
...
- * `PENDING` treatment. This is true even for "create"/"add" actions or
- * "delete" actions. A general rule of thumb to follow is that if a status
- * ends in -`ING`, it should get the animated yellow stripes of `PENDING`.
- *
- * @param {String} status The status to draw
- * @param {String} [api] Optionally specify which API mapping to use for the status
- * @param {String} [tooltip] The string to use for the tooltip. If omitted,
- *                           it will default to using the passed in status
- */
-.directive('rxStatusColumn', function (rxStatusMappings, rxStatusColumnIcons) {
```
Comment (CITguy) [inline src/rxStatusColumn/scripts/rxStatusMappings.js]: moved to `src/rxStatusColumn/scripts/rxStatusColumn.js`

File: src/rxStatusColumn/scripts/rxStatusMappings.js [hunk 2]
```diff
@@ -1,177 +1,4 @@
...
-})
-/**
- * @ngdoc object
- * @name rxStatusColumn.object:rxStatusColumnIcons
- * @description
- *
- * Mapping of internal statuses to FontAwesome icons.
- * The keys map to the names defined in rxStatusColumn.less
- */
-.value('rxStatusColumnIcons', {
```
Comment (CITguy) [inline src/rxStatusColumn/scripts/rxStatusMappings.js]: moved to `src/rxStatusColumn/scripts/rxStatusColumnIcons.js`

File: src/rxStatusColumn/scripts/rxStatusMappings.js [hunk 3]
```diff
@@ -1,177 +1,4 @@
...
- *         reverse="sort.reverse">
- *     </rx-sortable-column>
- * </th>
- * </pre>
- *
- * Note that `sort-property="status"` is referring to the `server.status`
- * property on each row. Thus the sorting is done in this example by the status
- * text coming from the API.
- */
-.directive('rxStatusHeader', function () {
```
Comment (CITguy) [inline src/rxStatusColumn/scripts/rxStatusMappings.js]: moved to `src/rxStatusColumn/scripts/rxStatusHeader.js`

File: src/rxStatusColumn/scripts/rxStatusMappings.spec.js [hunk 4]
```diff
@@ -1,151 +1,8 @@
 /* jshint node: true */
-
-describe('rxStatusColumn directive', function () {
```
Comment (CITguy) [inline src/rxStatusColumn/scripts/rxStatusMappings.spec.js]: moved to `src/rxStatusColumn/scripts/rxStatusColumn.spec.js`

**Comment by 100stacks** [comment]
LGTM

**Comment by bry4n** [comment]
LGTM

**Pull request merged**